include/variant_detection/bam_functions.hpp

```diff
@@ -5,6 +5,26 @@
 #include <seqan3/io/sam_file/output.hpp>
 #include <seqan3/utility/char_operations/predicate.hpp>
 
+
+/*!\details
+ * From the SAM/BAM Format Specification
+ * | Bit  | Bit   | Description                                                        | Note                                      |
+ * |:----:|:-----:|:------------------------------------------------------------------:|:-----------------------------------------:|
+ * |    1 | 0x1   | template having multiple segments in sequencing                    |                                           |
+ * |    2 | 0x2   | each segment properly aligned according to the aligner             | necessary for read pair method            |
```

Comment:
I do not agree with some of the notes. Here are a few suggestions:
- `0x1` - necessary for read pair method
- `0x2` - aligned as expected, probably not containing SVs
- `0x8` - mate unmapped, maybe an insertion?
- Flag `0x10` is not only important for split reads but also for paired-reads where `0x10` and `0x20` are compared. Usually you would expect them to be different and if they are not it could be because of an inversion. I don't know how to put this into a good concise note so you could also leave it empty.